external-api/api-tests/v1/email.api-test.ts

```diff
@@ -73,6 +90,21 @@ describe('/email', () => {
       expect(status).toEqual(unknown500Status);
       expect(body).toEqual({});
     });
+
+    describe('when GOV_NOTIFY_API_KEY is set to MOCK_E2E_GOV_NOTIFY_API_KEY', () => {
+      beforeEach(() => {
```

Comment:
An assertion where:
* `MOCK_E2E_GOV_NOTIFY_API_KEY` does not exist.
* `MOCK_E2E_GOV_NOTIFY_API_KEY` is ''.